spock-specs/src/test/groovy/org/spockframework/runtime/GroovyRuntimeUtilSpec.groovy

```diff
@@ -48,6 +49,28 @@ class GroovyRuntimeUtilSpec extends Specification {
     "setFoo"          | void       | null
   }
 
+  @Requires({ GroovyRuntimeUtil.MAJOR_VERSION <= 3 })
+  def "getterMethodToPropertyName Groovy <= 3"() {
+    expect:
+    GroovyRuntimeUtil.getterMethodToPropertyName(methodName, [], returnType) == propertyName
+
+    where:
+    methodName | returnType | propertyName
+    "isEmpty"  | Boolean    | "empty"
+    "isEmpty"  | boolean    | "isEmpty"
```

Comment:
Should the property name here be `empty` which probably is also the cause for the failing CI runs on Groovy 2.5 and 3.0?